antora/docs/modules/ROOT/pages/trusting_bundles.adoc

```diff
@@ -109,3 +109,58 @@ In order to add a new task to the existing set of Red Hat acceptable tasks, the
 * Once the pull request is submitted and approvevd, the CI process will automatically build the new task and add it to the list of acceptable tasks.
 ** You may view the acceptable bundles xref:{acceptable-bundles}[here].
 ** Additionally, you can view the task bundle in the https://quay.io/redhat-appstudio-tekton-catalog[Red Hat AppStudio Tekton Catalog]
+
+== Adding a custom task to the trusted task list
+
+Some users may want to add a custom build task to the default acceptable bundles list without pushing that task to the upstream acceptable task list. Below is a possible approach to make that happen.
+
+EC requires a customized policy in order to accept a custom task. Such a policy must include an additional data source with task bundle refs and digests relevant to the new custom task.
+
+For example, starting with https://github.com/enterprise-contract/config/blob/main/redhat-no-hermetic/policy.yaml[Red Hat's non-hermetic build policy], a new data source can be added under the `sources.data` key. See https://github.com/simonbaird/securesign-ec-config/blob/main/policy.yaml#L24[this policy] for a complete example.
+
+[,bash]
+---
+name: Secure Sign Custom Enterprise Contract Policy
+
+description: >-
+  Based on the standard Red Hat (non hermetic) policy
+  (see https://github.com/enterprise-contract/config/blob/main/redhat-no-hermetic/policy.yaml)
+  but with one additional data source. Currently this is an experimental POC,
+  see https://issues.redhat.com/browse/EC-147 .
```

Comment:
```suggestion
```

This commentary seems out of place for the official docs.

Reply:
+1, good catch. Removed this and the "experimental" bit from the above line.